zerver/webhooks/beeminder/view.py

```diff
@@ -0,0 +1,57 @@
+# Webhooks for external integrations.
+from typing import Text
+from django.http import HttpRequest, HttpResponse
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message, check_send_private_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile, get_user_profile_by_email
+import time
+current_time = time.time()
+
+@api_key_only_webhook_view("beeminder")
+@has_request_variables
+def api_beeminder_webhook(request, user_profile: UserProfile,
+                          payload=REQ(argument_type='body'),
+                          stream=REQ(default="beeminder"),
+                          email=REQ(default='foo@gmail.com'),
+                          topic=REQ(default='beekeeper')):
+
+    # type: (HttpRequest, UserProfile, Dict[str, Any], str, email, str) -> HttpResponse
```

Comment:
Okay, so this is a Python2 style mypy annotation. I would recommend using the Python3 syntax. [Here's an example](https://github.com/zulip/zulip/blob/master/zerver/webhooks/dropbox/view.py#L10) you can look at for further guidance! :)